app/js/ImageOptimiser.js

```diff
@@ -1,44 +1,41 @@
-/* eslint-disable
-    no-unused-vars,
-*/
-// TODO: This file was created by bulk-decaffeinate.
-// Fix any style issues and re-enable lint.
-/*
- * decaffeinate suggestions:
- * DS102: Remove unnecessary code created because of implicit returns
- * DS207: Consider shorter variations of null checks
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
- */
-const { exec } = require('child_process')
 const logger = require('logger-sharelatex')
-const Settings = require('settings-sharelatex')
+const metrics = require('metrics-sharelatex')
+const { callbackify } = require('util')
+const safeExec = require('./SafeExec').promises
```

Comment:
you can rename the function on the fly, right? because you named it differently in https://github.com/overleaf/filestore/pull/62/files#diff-db729691354748e8e0c733b256e0fee2R14

Reply:
That's correct, yes. :+1: A function is just an object, so you can assign it to any variable you like. :-)